.readthedocs.yaml

```diff
@@ -0,0 +1,14 @@
+version: 2
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "mambaforge-4.10"
+
+sphinx:
+  builder: html
+  configuration: docs/conf.py
+  fail_on_warning: false
```

Comment:
I prefer this be "true". Is there a reason not to?

Reply:
I will switch it on now